cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -417,28 +420,29 @@ traceChainMetrics
   :: forall blk. ()
   => HasHeader (Header blk)
   => Maybe EKGDirect
+  -> STM.TVar Word64
   -> BlockConfig blk
   -> ForgingStats
   -> Trace IO Text
   -> Tracer IO (ChainDB.TraceEvent blk)
-traceChainMetrics Nothing _ _ _ = nullTracer
-traceChainMetrics (Just _ekgDirect) _blockConfig _fStats tr = do
+traceChainMetrics Nothing _ _ _ _ = nullTracer
+traceChainMetrics (Just _ekgDirect) tForks _blockConfig _fStats tr = do
   Tracer $ \ev ->
     fromMaybe (pure ()) $ doTrace <$> chainTipInformation ev
   where
     chainTipInformation :: ChainDB.TraceEvent blk -> Maybe ChainInformation
     chainTipInformation = \case
       ChainDB.TraceAddBlockEvent ev -> case ev of
```

Comment:
If we generally trust that `SwitchedToAFork` is only emitted when it should be, then the changes to this case expression could be simpler; we just need to pass a `True` to `chainInformation` in one branch and a `False` in the other.